cmd/trace-agent/config/config.go

```diff
@@ -184,7 +184,7 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 	}
 
 	prevEnv := c.DefaultEnv
-	c.DefaultEnv = traceutil.NormalizeTag(c.DefaultEnv)
+	c.DefaultEnv = traceutil.NormalizeTag(c.DefaultEnv, false)
```

Comment:
We make a bunch of normalization calls for the env. I passed `false` to maintain the old behavior.
But aren't we just normalizing on the env's _value_? Shouldn't we pass `true`?